src/aap_eda/wsapi/consumers.py

```diff
@@ -452,7 +452,7 @@ def get_controller_info_from_aap_cred(
                         password=inputs.get("password", ""),
                     )
         except ObjectDoesNotExist:
-            logger.debug("AAP credential type not found")
+            logger.warning("AAP credential type not found")
```

Comment:
IMHO it would be helpful if this message had also the credential type; not a big issue though

Reply:
AAP is the credential type :) I added quotes to make it clearer.